src/test/java/io/github/externschool/planner/service/UserServiceTest.java

```diff
@@ -63,12 +69,20 @@ public void shouldReturnUserWhenCreateNewUser(){
     }
 
     @Test
-    public void shoudReturnUserWhenFindUserByEmail(){
+    public void shoudReturnUser_WhenFindUserByEmail(){
```

Comment:
We need to code style for tests and business methods :)

Reply:
Thinking about an issue label to fit best, 'bug' I guess, lol.